javascript/javascript2/week3/game.css

```diff
@@ -0,0 +1,47 @@
+body {
+    font-family: 'Times New Roman', Times, serif;
+    background-color: rgb(237, 232, 223)
+}
+
+h1 {
+    padding-left: 30rem;
+
+}
+
+.field {
+    margin-left: 10rem;
+    margin-right: 5rem;
+}
+
+* {
+    box-sizing: border-box;
+    color: rgb(79, 53, 5);
+}
+
+.msg,
+.start {
+    padding-left: 30rem;
+
+}
+
+#start,
+#duration {
+    padding: .5rem;
+    margin-bottom: 5rem;
+}
+
+section {
```

Comment:
It is in general a good practice to avoid styling tags directly but through the use of classes.

Reply:
👍🏻 I will